src/commands/apps-sdkconfig.ts

```diff
@@ -55,78 +60,78 @@ module.exports = new Command("apps:sdkconfig [platform] [appId]")
   )
   .option("-o, --out [file]", "(optional) write config output to a file")
   .before(requireAuth)
-  .action(
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    async (platform = "", appId = "", options: any): Promise<AppConfigurationData> => {
-      let appPlatform = getAppPlatform(platform);
-
-      if (!appId) {
-        let projectId = needProjectId(options);
-        if (options.nonInteractive && !projectId) {
-          throw new FirebaseError("Must supply app and project ids in non-interactive mode.");
-        } else if (!projectId) {
-          const result = await getOrPromptProject(options);
-          projectId = result.projectId;
-        }
+  .action(async (platform = "", appId = "", options: Options): Promise<AppConfigurationData> => {
+    let appPlatform = getAppPlatform(platform);
 
-        const apps = await listFirebaseApps(projectId, appPlatform);
-        // if there's only one app, we don't need to prompt interactively
-        if (apps.length === 1) {
-          appId = apps[0].appId;
-          appPlatform = apps[0].platform;
-        } else if (options.nonInteractive) {
-          throw new FirebaseError(
-            `Project ${projectId} has multiple apps, must specify an app id.`
-          );
-        } else {
-          const appMetadata: AppMetadata = await selectAppInteractively(apps, appPlatform);
-          appId = appMetadata.appId;
-          appPlatform = appMetadata.platform;
-        }
+    if (!appId) {
+      let projectId = needProjectId(options);
+      if (options.nonInteractive && !projectId) {
+        throw new FirebaseError("Must supply app and project ids in non-interactive mode.");
+      } else if (!projectId) {
+        const result = await getOrPromptProject(options);
+        projectId = result.projectId;
       }
 
-      let configData;
-      const spinner = ora(
-        `Downloading configuration data of your Firebase ${appPlatform} app`
-      ).start();
-      try {
-        configData = await getAppConfig(appId, appPlatform);
-      } catch (err: any) {
-        spinner.fail();
-        throw err;
+      const apps = await listFirebaseApps(projectId, appPlatform);
+      // Fail out early if there's no apps.
+      checkForApps(apps, appPlatform);
+      // if there's only one app, we don't need to prompt interactively
+      if (apps.length === 1) {
+        // If there's only one, use it.
+        appId = apps[0].appId;
+        appPlatform = apps[0].platform;
+      } else if (options.nonInteractive) {
```

Comment:
How does this behave when there are 0 apps? Do we show the correct error?

Reply:
Above, on L77, it checks for apps and will throw an error if there are no apps.

Reply:
Derp, just saw that you had a screenshot of this case. LGTM